ngrok-java-native/src/lib.rs

```diff
@@ -61,19 +61,27 @@ impl<'local> com_ngrok::RuntimeRs<'local> for RuntimeRsImpl<'local> {
                 RT.get_or_init(|| rt);
 
                 let jvm = self.env.get_java_vm().expect("cannot get jvm");
-                JVM.get_or_init(|| jvm);
 
                 let logref = self
                     .env
                     .new_global_ref(logger)
                     .expect("cannot get logger ref");
                 LOGGER.get_or_init(|| logref);
 
+                let jenv = jvm
```

Comment:
you already have an `env` in here, no need to attach a new one

Reply:
Ah, nice! Didn't realize those were the same type.